### PR TITLE
fix corrected qubit frequency after Ramsey fitting

### PR DESCRIPTION
**Pull request by DavidSarlle**
PR fixing issue detected in qibocal (https://github.com/qiboteam/qibocal/issues/337).
Now the delta_phys obtained in Ramsey routine is added or subtracted to the runcard qubit frequency  depending on the platform.

Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.

### CODE REVIEW

**Comment by DavidSarlle** [comment]
@andrea-pasquale this Pr also contains the latest qblox runcard from @aorgazf in order to test the changes

File: src/qibolab/platforms/abstract.py [hunk 1]
```diff
@@ -109,6 +109,8 @@ def __init__(self, name, runcard):
 
         self.single_qubit_natives = {}
         self.two_qubit_natives = TwoQubitNatives(0)
+        self._update_called_by_multiqubit = False
```
Comment (andrea-pasquale) [inline src/qibolab/platforms/abstract.py:112]: I don't like that fact that we need to add this method.
I think that we should be able to find a solution without adding this method.

File: src/qibolab/platforms/abstract.py [hunk 2]
```diff
@@ -237,6 +239,24 @@ def update(self, updates: dict):
...
 
+                # ramsey
+                elif par == "delta_frequency":
+                    delta_phys = int(value)
+                    qubit_freq = self.single_qubit_natives[qubit]["RX"]["frequency"]
+
+                    if self._update_called_by_multiqubit:
+                        corrected_qubit_frequency = int(qubit_freq - delta_phys)
+                    else:
+                        corrected_qubit_frequency = int(qubit_freq + delta_phys)
```
Comment (andrea-pasquale) [inline src/qibolab/platforms/abstract.py:250]: Same here.

**Review by andrea-pasquale — CHANGES_REQUESTED** [review]
Thanks @DavidSarlle.
You can find below some comments.
I don't like the attribute `_update_called_by_multiqubit`.
I think that a possible solution will be to move the call to `update` from `AbstractPlatform`
https://github.com/qiboteam/qibolab/blob/0aff24d1daa606904cc578814e0d37f273de79ab/src/qibolab/platforms/multiqubit.py#L152
to the top of the method.
In this way in `MultiqubitPlatform` you should be able to correct everything without having to do weird things.
Let me know if you agree.

**Comment by scarrazza** [comment]
@DavidSarlle given this PR is getting obsolete, we have decided to close and create a new one using the latest main branch.

**Pull request closed without merging**